README.md

```diff
@@ -1,2 +1,30 @@
-# factory-trytond
-Factory Boy - Trytond integration
+# Factory-trytond
+
+Factory-trytond is a [factory_boy](https://factoryboy.readthedocs.io/en/latest/introduction.html) extension developed to work with [Tryton ERP](https://www.tryton.org/). We can create our own ERP's model factories to do some testing or to populate our databases with some sample data.
+
+## How does it work?
+
+To use Factory-trytond your factory class only has to inherit the TrytonFactory:
+
+    import factory_trytond
+
+    class SaleFactory(factory_trytond.TrytonFactory):
+
+Once we have inherited the class, we can start configuring our factory like we would do with factory_boy (e.g: define the class the factory is referring to, the fields the factory has to work with).
```

Comment:
Split long lines with line returns, so that the markdown source is easy to read without line-wrapping readers. As long as the lines are consecutive markdown won't insert a paragraph break.